go.mod

```diff
@@ -1,19 +1,21 @@
 module github.com/kubernetes-csi/csi-lib-utils
 
-go 1.21
+go 1.22.0
```

Comment:
Should this be 1.22 instead?

Reply:
This is automatically generated. It also added the next line `toolchain go1.22.2`.